### PR TITLE
Flower Generation

### PR DESCRIPTION
**Pull request by Muqsit**
Generates random flowers in Plains and Blue Orchids in Swamps...just like Minecraft :3

### CODE REVIEW

**Comment by Muqsit** [comment]
Tested.

File: src/pocketmine/level/generator/normal/biome/SwampBiome.php [hunk 1]
```diff
@@ -20,23 +20,26 @@
...
+		
+		$this->addPopulator($flower);
 		$this->setElevation(62, 63);
-
 		$this->temperature = 0.8;
 		$this->rainfall = 0.9;
 	}
-
-	public function getName() : string{
+	public function getName(){
```
Comment (xpyctum) [inline src/pocketmine/level/generator/normal/biome/SwampBiome.php]: Why remove : string ?

Comment (Muqsit) [inline src/pocketmine/level/generator/normal/biome/SwampBiome.php]: I modified it to my server's needs. One of the plugins was spamming the console with an error associated with it. :/

**Comment by xpyctum** [comment]
You should add : string for php7, because if i start this from php7 u get a crash.

**Comment by RandomAltThingy** [comment]
You guys here should **not** merge pull requests if there is too many commits. People should learn to use Git Bash.

**Comment by Muqsit** [comment]
I'm newbie sorry.

**Comment by Muqsit** [comment]
Delete please..

**Comment by RandomAltThingy** [comment]
1) You cannot delete Pull Requests
2) I didn't say close it, I was making a suggestion to the Genisys devs

**Comment by RandomAltThingy** [comment]
Also make sure the build compiles on your next PR

**Comment by Muqsit** [comment]
Thanks, I'm just a starter btw, not a professional like you and peratx :-:

**Comment by Muqsit** [comment]
I use wanted to delete this because I had no intention of adding the taiga Biome thingy in this pr

**Comment by RandomAltThingy** [comment]
I wouldn't call me a "pro", I'm only 13 :P. But I'm good if I don't say so myself.

**Comment by Muqsit** [comment]
:P

**Comment by PeratX** [comment]
@Muqsit Just create another pull request to merge this in one or two commits

**Pull request closed without merging**